src/explore-education-statistics-admin/src/pages/publication/components/PublicationContactForm.tsx

```diff
@@ -0,0 +1,133 @@
+import Button from '@common/components/Button';
+import ButtonGroup from '@common/components/ButtonGroup';
+import ButtonText from '@common/components/ButtonText';
+import { FormFieldset } from '@common/components/form';
+import Form from '@common/components/form/Form';
+import FormFieldTextInput from '@common/components/form/FormFieldTextInput';
+import ModalConfirm from '@common/components/ModalConfirm';
+import useFormSubmit from '@common/hooks/useFormSubmit';
+import useToggle from '@common/hooks/useToggle';
+import Yup from '@common/validation/yup';
+import { Formik } from 'formik';
+import React from 'react';
+
+export interface FormValues {
+  teamName: string;
+  teamEmail: string;
+  contactName: string;
+  contactTelNo: string;
+}
+
+interface Props {
+  initialValues: FormValues;
+  onCancel: () => void;
+  onSubmit: (values: FormValues) => void;
+}
+
+const PublicationContactForm = ({
+  initialValues,
+  onCancel,
+  onSubmit,
+}: Props) => {
+  const [showConfirmSubmitModal, toggleShowConfirmSubmitModal] = useToggle(
+    false,
+  );
+
+  const validationSchema = Yup.object<FormValues>({
+    teamName: Yup.string().required('Enter a team name'),
+    teamEmail: Yup.string()
+      .required('Enter a team email')
+      .email('Enter a valid team email'),
+    contactName: Yup.string().required('Enter a contact name'),
+    contactTelNo: Yup.string().required('Enter a contact telephone'),
+  });
+
+  return (
+    <Formik<FormValues>
+      enableReinitialize
+      initialValues={{
+        ...(initialValues ?? {
+          teamName: '',
+          teamEmail: '',
+          contactName: '',
+          contactTelNo: '',
+        }),
+      }}
+      validationSchema={validationSchema}
+      onSubmit={useFormSubmit((values: FormValues) => {
+        onSubmit(values);
+      })}
+    >
+      {form => (
+        <>
+          <Form id="publicationContactForm">
+            <FormFieldset
+              id="contact"
+              legend="Contact for this publication"
+              legendSize="m"
```

Comment:
The size of this is different to the read-only mode. Would at least make them the same size to be consistent.

We might want to consider just removing this fieldset and just use the heading and help text in both read-only and editable modes.

Reply:
I've gone for removing the fieldset as it's not really adding anything.